### PR TITLE
Update Skroutz Analytics Privacy Policy

### PR DESCRIPTION
**Pull request by MrWhizzy**
We need to update the [Skroutz Analytics Privacy Policy section](http://developer.skroutz.gr/analytics/privacy_policy/) in order to be GDPR and ePrivacy compliant.

Also, a relevant page regarding the CPS Privacy Policy needs to be added as well.

### CODE REVIEW

File: source/localizable/analytics/privacy_policy.html.md.erb [hunk 1]
```diff
@@ -15,143 +15,201 @@ title: privacy_policy
...
+| _sa_meta          | Skroutz Analytics - Ανάλυση της διάδρασης των χρηστών της ιστοσελίδας, κατόπιν της μετάβασής τους από τη Skroutz.gr. Σκοπός είναι η αξιολόγηση και βελτίωση των υπηρεσιών μας, καθώς και των υπηρεσιών Skroutz, μέσω σχετικής στατιστικής ανάλυσης χρήσης. | Μέχρι το κλείσιμο του φυλλομετρητή | Skroutz                       |
+| __skr_nltcs_mt    | Skroutz Analytics - Ανάλυση της διάδρασης των χρηστών της ιστοσελίδας, κατόπιν της μετάβασής τους από τη Skroutz.gr. Σκοπός είναι η αξιολόγηση και βελτίωση των υπηρεσιών μας, καθώς και των υπηρεσιών Skroutz, μέσω σχετικής στατιστικής ανάλυσης χρήσης. | Μέχρι το κλείσιμο του φυλλομετρητή | Skroutz                       |
+| sa_session        | Skroutz Analytics - Ανάλυση της διάδρασης των χρηστών της ιστοσελίδας, κατόπιν της μετάβασής τους από τη Skroutz.gr. Σκοπός είναι η αξιολόγηση και βελτίωση των υπηρεσιών μας, καθώς και των υπηρεσιών Skroutz, μέσω σχετικής στατιστικής ανάλυσης χρήσης. | 1 μήνας                            | Skroutz                       |
```
Comment (kosdiamantis) [inline source/localizable/analytics/privacy_policy.html.md.erb]: There is a typo  here, it's `_sa_session`

Also the ttl of this cookie is 30 minutes, and the description should be the same as `analytics_session` and `__skr_nltcs_ss`.

🤔 I think `Skroutz Analytics -` is redundant for the `_sa_meta`, `__skr_nltcs_mt` and `_sa_session`.

Comment (kosdiamantis) [reply source/localizable/analytics/privacy_policy.html.md.erb]: ... or perhaps we should add this prefix to all the Skroutz Analytics' cookies

File: source/localizable/analytics/cps_privacy_policy.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+- Η διεύθυνση IP του χρήστη
+- Στοιχεία που αφορούν στην εκάστοτε συσκευή (όπως η έκδοση του λειτουργικού
+  συστήματος, η σύνδεση μέσω app ή web, η έκδοση του προγράμματος περιήγησής
+  της)
+
+Τα παραπάνω συνδέονται με αναγνωριστικό παραγγελίας, από το οποίο δεν προκύπτει
+ευθέως ταυτοποίηση του χρήστη για τον Skroutz. Για λοιπές κατηγορίες δεδομένων
+που συλλέγονται από το Skroutz όταν είναι ενεργοποιημένη παράλληλα και η
+υπηρεσία Skroutz Analytics, δείτε αναλυτικότερα
+[εδώ](https://developer.skroutz.gr/analytics/privacy_policy/).
```
Comment (kosdiamantis) [inline source/localizable/analytics/cps_privacy_policy.html.md.erb]:
```markdown
[εδώ](/analytics/privacy_policy/)
```

File: source/localizable/analytics/cps_privacy_policy.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+τους χρήστες.
+
+Στο πλαίσιο αυτό, ενημερώνουμε λεπτομερώς τους χρήστες για την επεξεργασία των
+προσωπικών τους δεδομένων για συναλλακτικούς σκοπούς, τόσο στην [πολιτική
+απορρήτου](https://www.skroutz.gr/privacy) καθώς και την [πολιτική
+cookies](https://www.skroutz.gr/cookies) του Skroutz.gr.
+
+Αναφορικά με τη νομιμότητα της επεξεργασίας που αφορά στη χρήση του Skroutz
+Analytics, παρακαλώ δείτε περισσότερα στην [Πολιτική Απορρήτου Skroutz
+Analytics](https://developer.skroutz.gr/analytics/privacy_policy/).
```
Comment (kosdiamantis) [inline source/localizable/analytics/cps_privacy_policy.html.md.erb]:
```markdown
[Πολιτική Απορρήτου Skroutz Analytics](/analytics/privacy_policy/)
```

File: source/localizable/analytics/privacy_policy.html.md.erb [hunk 4]
```diff
@@ -15,143 +15,201 @@ title: privacy_policy
...
+τις απαιτήσεις της νομοθεσίας, αλλά και τις οδηγίες της Ελληνικής Αρχής
+Προστασίας Δεδομένων Προσωπικού Χαρακτήρα για λήψη έγκυρης, ενημερωμένης
+συγκατάθεσης του χρήστη και με βάση όλα τα ανωτέρω, μπορεί κατά περίπτωση να
+χρησιμοποιηθεί από τα καταστήματα ελεύθερα.
+
+**Δεν απαιτείται νομικά επιπλέον συγκατάθεση (double consent), μέσω της
+ιστοσελίδας του εκάστοτε εμπόρου.** Περισσότερες πληροφορίες για τη νομιμότητα
+της επεξεργασίας μπορείτε να βρείτε εδώ.
+
+[TODO]: # (Where is `εδώ`?)
```
Comment (kosdiamantis) [inline source/localizable/analytics/privacy_policy.html.md.erb]: It's not here :trollface:

Comment (MrWhizzy) [reply source/localizable/analytics/privacy_policy.html.md.erb]: 🧐

File: source/localizable/analytics/cps_privacy_policy.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+τους χρήστες.
+
+Στο πλαίσιο αυτό, ενημερώνουμε λεπτομερώς τους χρήστες για την επεξεργασία των
+προσωπικών τους δεδομένων για συναλλακτικούς σκοπούς, τόσο στην [πολιτική
+απορρήτου](https://www.skroutz.gr/privacy) καθώς και την [πολιτική
+cookies](https://www.skroutz.gr/cookies) του Skroutz.gr.
+
+Αναφορικά με τη νομιμότητα της επεξεργασίας που αφορά στη χρήση του Skroutz
+Analytics, παρακαλώ δείτε περισσότερα στην [Πολιτική Απορρήτου Skroutz
+Analytics](/analytics/privacy_policy/#section-7).
```
Comment (harrisred) [inline source/localizable/analytics/cps_privacy_policy.html.md.erb]: I think this should be `#section-8`

Comment (MrWhizzy) [reply source/localizable/analytics/cps_privacy_policy.html.md.erb]: indeed

**Pull request merged**